src/static/simple.css

```diff
@@ -721,4 +721,56 @@ sub {
   margin: 2rem 0;
   overflow-x: auto;
   padding: 1.5rem;
-}
\ No newline at end of file
+}
+
+.form-group{
```

Comment:
Nitpick: Could we auto-format this file? Maybe we could add the auto-formatting of CSS and HTML to our check script? It's outside the scope of this PR though. I can open a PR for this.